build-gluster-org/scripts/gh_centos-regression.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+MY_ENV=$(env | sort)
+BURL=${BUILD_URL}consoleFull
+
+function vote_gerrit() {
+    VOTE="$1"
+    VERDICT="$2"
+    OUTPUT=$(echo ; cat $3)
+    echo "$VOTE '$BURL : $VERDICT \n$OUTPUT'"
+}
+
+# Display all environment variables in the debugging log
+echo "Start time $(date)"
+echo
+echo "Display all environment variables"
+echo "*********************************"
+echo
+echo "$MY_ENV"
+echo
+
+# use "7 and not "7" since RHEL use 7.6 while Centos use 7
+grep -q 'VERSION_ID="7' /etc/os-release && export PYTHON=/usr/bin/python2.7
+
+# FB and experimental branch gets a pass
+if [ "$GERRIT_BRANCH" = "release-3.8-fb" ] ||  [ "$GERRIT_BRANCH" = 'experimental' ]; then
```

Comment:
I think that part can be removed for github

Reply:
Done